docs/pages/projectExplorer/run-builds-compiles-and-actions.md

```diff
@@ -6,17 +6,29 @@ The build or compiles of a project are based on the `buildCommand` and `compileC
 
 Before running a build, the project's build command must be set. This can be done using the **Set Build Command** action on the project. Here you can leverage any build tool such as `elias` in ARCAD Builder or `makei` in `ibmi-bob`.
 
-![Set Build Command](../../assets/ProjectExplorer_11.png)
+![Set Build Command](../../assets/ProjectExplorer_11.png ':size=500')
 
 To then launch a build, use the **Run Build** action on the project heading or the `ctrl+shift+b` shortcut (`cmd+shift+b` on Mac). This will first deploy the project and then invoke the build command. To view the output of this build process, navigate to the **Output** view and select the **IBM i Output** channel.
 
 ![Run Build](../../assets/ProjectExplorer_12.png)
 
+In the case you want to simply build a specific object in your project, you can set your project's build object command using the **Set Build Object Command** action on the project.
+
+![Set Build Object Command](../../assets/ProjectExplorer_41.png ':size=500')
+
+To then build a target object, locate the object under the **Library List** or **Object Libraries** and use the right-click **Run Build Object** action.
```

Comment:
I think the menu item would be better as 'Build Object', this is different than 'Run Build' which implies running the full build.  I know this means redoing some screen caps, but I believe it would be an improvement